stubs/inertia-vue/resources/js/Components/TextInput.vue

```diff
@@ -1,14 +1,7 @@
 <script setup>
 import { onMounted, ref } from 'vue';
 
-defineProps({
-    modelValue: {
-        type: String,
-        required: true,
-    },
-});
-
-defineEmits(['update:modelValue']);
+const modelValue = defineModel();
```

Comment:
```suggestion
const modelValue = defineModel({
    type: String,
    required: true,
});
```
I do not use Vue much, and this is untested and just from reading the docs: https://vuejs.org/api/sfc-script-setup.html#definemodel